manifests/reload_service.pp

```diff
@@ -19,12 +19,16 @@
       $http_addr = $::consul::http_addr
     }
 
+    case $::consul::install_method {
+      'docker': { $command = "docker exec -t consul consul reload -http-addr=${http_addr}:${consul::http_port}" }
```

Comment:
That's a good question.  I have it stuck in my head that's run into issues without the TTY enabled ... but honestly this could be based on earlier releases/info.  Ill give a test internally and see what's up with that.